backend/models/Books.js

```diff
@@ -1,70 +1,64 @@
-import { Schema, model, SchemaTypes } from "mongoose";
+import { Schema, model, SchemaTypes } from 'mongoose';
 
 const bookSchema = new Schema({
-  title: {
-    type: String,
-    required: [true, "title is required"],
-    trim: true,
-    minlength: 3,
-  },
-  description: {
-    type: String,
-    required: [true, "description is required"],
-    trim: true,
-    minlength: 20,
-  },
-  // Bia: Added to have a static image URL for book cover
-  imageUrl: {
-    type: String,
-    trim: true,
-  },
-  genre: {
-    type: String,
-    required: [true, "genre is required"],
-    enum: [
-      "Fiction",
-      "Non-Fiction",
-      "Science Fiction",
-      "Fantasy",
-      "Biography",
-      "History",
-      "Romance",
-      "Thriller",
-      "Mystery",
-      "Self-Help",
-      "Health",
-    ],
-    trim: true,
-  },
-  userId: {
-    type: SchemaTypes.ObjectId,
-    ref: "User",
-    required: true,
-    alias: "author",
-  },
-  isBan: {
-    type: String,
-    required: [true, "isBan is required"],
-    trim: true,
-    unique: true,
-    minlength: 10,
-    maxlength: 13,
-  },
-  created_at: {
-    type: Date,
-    default: () => Date.now(),
-    immutable: true,
-  },
-  updated_at: { type: Date },
+	title: {
+		type: String,
+		required: [true, 'title is required'],
+		trim: true,
+		minlength: 3,
+	},
+	description: {
+		type: String,
+		required: [true, 'description is required'],
+		trim: true,
+		minlength: 20,
+	},
+	genre: {
+		type: String,
+		required: [true, 'genre is required'],
+		enum: [
+			'Fiction',
+			'Non-Fiction',
+			'Science Fiction',
+			'Fantasy',
+			'Biography',
+			'History',
+			'Romance',
+			'Thriller',
+			'Mystery',
+			'Self-Help',
+			'Health',
+		],
+		trim: true,
+	},
+	author: {
+		type: String,
+		required: [true, 'author is required'],
+		minlength: 6,
+	},
+	isBan: {
```

Comment:
I think `isBan` can be removed. @alexh205

Reply:
The 'isBan' property has been updated to 'ISBN'.